lib/inquirer-prompt.js

```diff
@@ -15,38 +15,16 @@ const chalk = require('chalk');
 
 module.exports = function prompt(questions, config) {
 	inquirer.prompt(questions).ui.process
-	.reduce(function (newOpts, ans) {
-		return attachAnswers(newOpts, ans);
-	}, Object.assign(config, require('./utils/initial-config')))
-	.flatMap(newOpts => {
-		const parser = require('./parser/index');
-		return parser(null, checkEmptyAnswers(newOpts));
-	})
-	.subscribeOnError( (err) => process.stdout.write(err.toString()));
-};
-
-/*
-* @function attachAnswers
-*
-* Adds the answers from the inquirer instance if the type of the question is
-* equal to the configuration property
-*
-* @param { Object } config - initial questions provided from --init
-* @param { Object } answers - The answers passed from the inquirer instance
-* @returns { Object } newConfig - An new object with the answers added to it
-*/
+	.subscribe(ans => {
+		config = require(`./init-transforms/${ans.name}`)(config, ans.answer);
 
-function attachAnswers(config, answers) {
-	let newConfig;
-	Object.keys(config).forEach( (configName) => {
-		if(configName == answers.name) {
-			newConfig = Object.assign(config, {
-				[configName]: answers.answer
-			});
-		}
+	}, err => process.stdout.write(err.toString()),
+	() => {
+		console.log(chalk.white('We managed to generate a webpack config for you.'));
```

Comment:
I'd just write:

`✓ Webpack config generated! Please review it` and I'd use `chalk.green` for it.